<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailMode</class>
 <widget class="QWidget" name="DetailMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1508</width>
    <height>733</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DetailMode{
background: #5C5F66;
}
*{
color: white;
box-sizing: border-box;
}
#Start, #upload{
	background: #18453B;
	transition-duration: 1s;
	cursor: pointer;
	color: white;
	font-weight: 500;
	border: 1px solid #003300;
}
#Start:hover{
background: #538efc;
}
#upload:hover{
background: #538efc;
}
.QSpinBox, QDoubleSpinBox{
max-width: 60px;
border: none;
outline: none;
background: white;
color: black;
border: 1px solid #18453B;
padding: 4px;
}
.QFrame{
background:#2C2E33;
border-radius: 4px
}
.QGroupBox{
border: none;
font-size: 16px;
font-weight: 700;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1511</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Детальный режим построения РЛИ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bsuir 2023, Minsk</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Путь:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Path">
         <property name="styleSheet">
          <string notr="true">color: #7b7c88</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="file_name_label">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Файл ТС:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="File_name">
         <property name="styleSheet">
          <string notr="true">color: #7b7c88</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="upload">
       <property name="styleSheet">
        <string notr="true">height: 32px</string>
       </property>
       <property name="text">
        <string>Загрузить CONSORT файл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>401</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Коэффициент передискретизации: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="Kss">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Дискретность по долготе, м: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dxsint">
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Дискретность по широте, м: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dysint">
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Число точек по долготе: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="Nxsint">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>201</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Число точек по широте: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="Nysint">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>201</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Однопроходная интерферометрия</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>801</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="TypeWinDn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>361</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Весовая функция по наклонной дальности</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>296</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="r1">
        <property name="text">
         <string>нет</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r2">
        <property name="text">
         <string>косинус</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r3">
        <property name="text">
         <string>косинус квадрат</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r4">
        <property name="text">
         <string>Хемминга</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r5">
        <property name="text">
         <string>Хемминга (третья степень)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r6">
        <property name="text">
         <string>Хемминга (четвертая степень)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r7">
        <property name="text">
         <string>Кайзера-Бесселя, alfa=2.7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r8">
        <property name="text">
         <string>Кайзера-Бесселя, alfa=3.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r9">
        <property name="text">
         <string>Кайзера-Бесселя, alfa=3.5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r10">
        <property name="text">
         <string>Блекмана-Херриса</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TypeWinDp">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>361</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Весовая функция по поперечной дальности</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>296</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QRadioButton" name="p1">
        <property name="text">
         <string>нет</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="p2">
        <property name="text">
         <string>косинус</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="p3">
        <property name="text">
         <string>косинус квадрат</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="p4">
        <property name="text">
         <string>Хемминга</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="p5">
        <property name="text">
         <string>Хемминга (третья степень)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="p6">
        <property name="text">
         <string>Хемминга (четвертая степень)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="p7">
        <property name="text">
         <string>Кайзера-Бесселя, alfa=2.7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="p8">
        <property name="text">
         <string>Кайзера-Бесселя, alfa=3.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="p9">
        <property name="text">
         <string>Кайзера-Бесселя, alfa=3.5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="p10">
        <property name="text">
         <string>Блекмана-Херриса</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>401</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QCheckBox" name="FlagViewSignal">
       <property name="text">
        <string>Отображать сигналы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="GPU">
       <property name="text">
        <string>Рассчет с использованием драйвера CUDA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="FlagWriteRli">
       <property name="text">
        <string>Сохранить результат</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Задержка момента построения РЛИ-1, с</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="tauRli"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Время синтезирования, c</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Tsint">
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.400000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Степень яркостного изображения: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="StepBright">
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>640</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>РАССЧИТАТЬ</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserModalDate">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>500</y>
     <width>601</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
border-radius: 3px solid grey</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>70</y>
     <width>261</width>
     <height>541</height>
    </rect>
   </property>
   <property name="title">
    <string>Terminal</string>
   </property>
   <widget class="QTextBrowser" name="Output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black;
color: green;
font: 11pt &quot;SimSun-ExtB&quot;;
border: 1px solid white;</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="RowsAndCulCount">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>500</y>
     <width>191</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
border-radius: 3px solid grey</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
